topics/groovy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: groovy-lang
+created_by: James Strachan
+display_name: Groovy
+logo: groovy.png
+released: January 2, 2007
```

Comment:
Where did you get this release date? The Wikipedia page has 2003 as when Groovy first appeared.

You could add a `related: grails, java, language` line.

Reply:
I was going with `Groovy 1.0 was released on January 2, 2007` from the wiki. I'll change it even though there's no mention of a "release" tied to the 2003 date.